### ISSUE TITLE
coverage directory currently included for configPkg

### ISSUE THREAD

**Issue by ThisIsMissEm** [adonisjs/eslint-config#7]
### Package version

^2.0.0-beta.7

### Describe the bug

When running `npm run lint` (`eslint .`) the following output is given:
```
package/coverage/block-navigation.js
  1:1  warning  Unused eslint-disable directive (no problems were reported)

package/coverage/prettify.js
  1:1  warning  Unused eslint-disable directive (no problems were reported)

package/coverage/sorter.js
  1:1  warning  Unused eslint-disable directive (no problems were reported)
```

It seems the coverage directory, despite being gitignored is still included in the configPkg defaults to check.

(Note: reproduction repo hasn't yet been upgraded)

### Reproduction repo

https://github.com/ThisIsMissEm/adonisjs-respond-with

**Comment by ThisIsMissEm** [comment]
Seems that the file include list of `**/*.ts` some how ends up including `**/*.js` which is a bit weird.

**Comment by ThisIsMissEm** [comment]
Seems to have come from using `eslint .`, switching to `"lint": "eslint '**/*.ts'",` gives no errors as expected.

**Comment by thetutlage** [comment]
Yup, I think we should add the coverage directory to the `ignores` list. Can you send a PR for it?

**Comment by ThisIsMissEm** [comment]
It appears there already but didn't seem to work, it may have been because I had eslintConfig in my package.json or because of how I was invoking eslint, but after this change it seems to be working fine: https://github.com/ThisIsMissEm/adonisjs-respond-with/commit/89c2482f43e74c1a9735886ab2f9bd646f2b2c26

**Comment by thetutlage** [comment]
This change breaks the behavior. https://github.com/adonisjs/eslint-config/commit/c7407f103f56932577ab893e295bc552bbc24c94#diff-e727e4bdf3657fd1d798edcd6b099d6e092f8573cba266154583a746bba0f346R228

At this stage, I have no idea where to keep things configs to have the desired outcome. Lemme try to read ESLint docs one more time and see if I can make sense of it

**Comment by thetutlage** [comment]
Should be fixed by this https://github.com/adonisjs/eslint-config/commit/7479dae9ac3fbcdfc0fef482c0246f27713cbfaa
